File: objectModel/Python/cdm/persistence/cdmfolder/types/projections/operation_add_attribute_group.py
# Copyright (c) Microsoft Corporation. All rights reserved.
# Licensed under the MIT License. See License.txt in the project root for license information.

from typing import Optional

from cdm.persistence.cdmfolder.types.projections.operation_base import OperationBase


class OperationAddAttributeGroup(OperationBase):
    """OperationAddAttributeGroup class"""

    def __init__(self):
        super().__init__()

        self.attributeGroupName = None  # type: Optional[str]

File: objectModel/Java/objectmodel/src/main/java/com/microsoft/commondatamodel/objectmodel/persistence/cdmfolder/types/projections/OperationAddAttributeGroup.java
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License. See License.txt in the project root for license information.

package com.microsoft.commondatamodel.objectmodel.persistence.cdmfolder.types.projections;

/**
 * OperationAddAttributeGroup class
 */
public class OperationAddAttributeGroup extends OperationBase {
    private String attributeGroupName;

    public String getAttributeGroupName() {
        return attributeGroupName;
    }

    public void setAttributeGroupName(String attributeGroupName) {
        this.attributeGroupName = attributeGroupName;
    }
}
